src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -142,43 +142,50 @@ const ExerciseFinishedScreen = ({ navigation, route }: Props): ReactElement => {
   const { message, resultColor, buttonText, ResultIcon, navigationAction } = helper()
 
   return (
-    <SafeAreaView style={{ flex: 1 }}>
-      <Root>
-        <UpperSection unlockedNextExercise={unlockedNextExercise}>
-          <Icon onPress={() => navigation.dispatch(closeExerciseAction)}>
-            {unlockedNextExercise ? (
-              <CloseIcon width={wp('6%')} height={wp('6%')} />
-            ) : (
-              <CloseIconWhite width={wp('6%')} height={wp('6%')} />
-            )}
-          </Icon>
-          <ResultIcon width={wp('10%')} height={wp('10%')} />
-          <MessageContainer>
-            <Message unlockedNextExercise={unlockedNextExercise}>{message}</Message>
-            <Results color={resultColor}>
-              {correctResults.length} {labels.results.of} {results.length} {labels.general.words}{' '}
-              {labels.results.correct}
-            </Results>
-            <Progress.Bar
-              color={resultColor}
-              progress={percentageOfCorrectResults}
-              unfilledColor={theme.colors.background}
-              width={wp('40%')}
-              height={wp('2%')}
-              borderWidth={0}
-            />
-          </MessageContainer>
-        </UpperSection>
+    <>
+      <RouteWrapper
+        separated
+        backgroundColor={unlockedNextExercise ? theme.colors.correct : theme.colors.primary}
```

Comment:
This somehow does not work for me, the status bar is always white. Or is this on purpose?

Reply:
if you unlocked a new exercise it should be green. gonna check it again

Reply:
pls check screenshot of other conversation here

Reply:
https://github.com/digitalfabrik/lunes-app/pull/329#discussion_r935394187